File: src/ClientApp/src/app/pages/leads/view-models/lookup.view-model.ts
export interface LookupViewModel {
    id: string;
    name: string;
    code?: string;
    dialingCode?: string;
    partnerTypeCode?: string;
    productIds?: string[];
    ownershipTypeCode?: string;
    ownerPartnerId?: string | null;
    canOwnProducts?: boolean;
    canSellInHouseProducts?: boolean;
}

File: src/API/src/Modules/Partners/Partners.Application/ViewModels/LookupViewModel.cs
namespace Partners.Application.ViewModels
{
    public class LookupViewModel
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string? Code { get; set; }
        public string? PartnerTypeCode { get; set; }
        public bool? CanOwnProducts { get; set; }
        public bool? CanSellInHouseProducts { get; set; }
        public List<Guid> ProductIds { get; set; } = new();
    }
}
